src/waveform/vsyncthread.h

```diff
@@ -42,41 +42,35 @@ class VSyncThread : public QThread {
         ST_COUNT // Dummy Type at last, counting possible types
     };
 
-    static void swapGl(QGLWidget* glw, int index);
+    static void swapGl(QOpenGLWidget* glw, int index);
 
     VSyncThread(QObject* pParent, GuiTick* pGuiTick);
     ~VSyncThread();
 
     void run();
     void stop();
 
-    bool waitForVideoSync(QGLWidget* glw);
     int elapsed();
     int toNextSyncMicros();
     void setSyncIntervalTimeMicros(int usSyncTimer);
     void setVSyncType(int mode);
     int droppedFrames();
-    void setSwapWait(int sw);
     int fromTimerToNextSyncMicros(const PerformanceTimer& timer);
     void vsyncSlotFinished();
     void getAvailableVSyncTypes(QList<QPair<int, QString > >* list);
-    void setupSync(QGLWidget* glw, int index);
-    void waitUntilSwap(QGLWidget* glw);
 
   signals:
     void vsyncRender();
     void vsyncSwap();
 
   private:
     bool m_bDoRendering;
-    //QGLWidget *m_glw;
```

Comment:
Please remove code rather than commenting it out.

Reply:
I did, it was already commented out there :-)

Reply:
Oops, sorry I misread the diff.